Fix: guard against webassembly issue (#628)
MetaMask/snap-bitcoin-wallet: packages/snap/CHANGELOG.md

- Add `signProofOfOwnership` client request method ([#626](https://github.com/MetaMask/snap-bitcoin-wallet/pull/626))
  - This method silently signs `metamask:proof-of-ownership:<nonce>:<address>` messages with the account's BIP-322 signer

### Fixed

> - Throw a clear `WalletError` when WebAssembly is unavailable (e.g. iOS Lockdown Mode) instead of crashing with an obscure `"undefined is not an object (evaluating 'WebAssembly.instantiate')"` error ([#628](https://github.com/MetaMask/snap-bitcoin-wallet/pull/628))

Reviewer: nit:
```suggestion
- Throw a `WalletError` when `WebAssembly` is unavailable (e.g. iOS Lockdown Mode) ([#628](https://github.com/MetaMask/snap-bitcoin-wallet/pull/628))
  - The snap was previously crashing with a confusing `"undefined is not an object (evaluating 'WebAssembly.instantiate')"` error.
```
